src/data.json

```diff
@@ -0,0 +1,59 @@
+{
+  "captionTitle": "The MONOLOGUE travel guide series",
+  "title": "Japanese Ilustration",
+  "image": "../src/images/back.jpg",
+  "legend": "this is where the legend should be",
+  "chapters": [
+    {
+      "chapterTitle": "Chapter 1",
+      "chapterIcon": "1"
+    },
+    {
+      "chapterTitle": "Chapter 2",
+      "chapterIcon": "2"
+    },
+    {
+      "chapterTitle": "Chapter 3",
+      "chapterIcon": "3"
+    },
+    {
+      "chapterTitle": "Chapter 4",
+      "chapterIcon": "4"
+    },
+    {
+      "chapterTitle": "Chapter 5",
+      "chapterIcon": "5"
+    },
+    {
+      "chapterTitle": "Chapter 6",
+      "chapterIcon": "6"
+    },
+    {
+      "chapterTitle": "Chapter 7",
+      "chapterIcon": "7"
+    },
+    {
+      "chapterTitle": "Chapter 8",
+      "chapterIcon": "8"
+    },
+    {
+      "chapterTitle": "Chapter 9",
+      "chapterIcon": "9"
+    },
+    {
+      "chapterTitle": "Chapter 10",
+      "chapterIcon": "10"
+    }
+  ],
+  "subtitles": [
+    {
+        "title": "Subtitle 1"
```

Comment:
I think it might be confusing using title in here since subtitle is a lower title category than title, maybe you can use text or content or if you are not planning to add other props to these objects you can just use something like: 
` "subtitles": ["Subtitle 1", "Subtitle 2", ...]`